examples/runHscQuickTest.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# USES DATA IN ci_hsc
+
+# Setup
+export OMP_NUM_THREADS=1
+
+# Fake out the pipeline about the origin of the reference catalog
+export SETUP_ASTROMETRY_NET_DATA="astrometry_net_data sdss-dr9-fink-v5b"
+export ASTROMETRY_NET_DATA_DIR=${CI_HSC_DIR}/sdss-dr9-fink-v5b
+
+REPO='data_hsc_quick'
+RERUN=20170105
+
+PRODUCT_DIR="${VALIDATE_DRP_DIR}"
+
+CAMERA=HscQuick
+YAMLCONFIG="${PRODUCT_DIR}"/examples/"${CAMERA}".yaml
+
+# DATA_DIR=${VALIDATION_DATA_HSC_DIR}
+DATA_DIR=${CI_HSC_DIR}
+CALIB_DIR=${CI_HSC_DIR}/CALIB
+
+# Ingest raw data into repo
+mkdir -p ${REPO}
+ln -s ${CALIB_DIR} ${REPO}/CALIB
```

Comment:
It is a good idea to enclose command arguments with `""` to prevent word splitting (some crazy people but spaces in paths).